docs/solutions/runtime-errors/liberal-boolean-xml-parsing-opnsense-pfsense.md

```
@@ -0,0 +1,230 @@
+  - docs/solutions/architecture-issues/pkg-internal-import-boundary.md
+  - docs/plans/2026-04-18-002-fix-issue-558-parser-on-value.md
+  - GOTCHAS.md#15-liberal-boolean-and-integer-parsing
+---
+
+# Liberal Boolean Parsing for OPNsense/pfSense config.xml
+
+## Problem
+
+`opndossier audit <config.xml>` crashed on an OPNsense 26.1 `config.xml` with:
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)